app/scripts/views/sign_up.js

```diff
@@ -48,6 +48,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     beforeRender: function () {
+      var self = this;
+      require(['passwordcheck'], function (passwordcheck) {
```

Comment:
This is an asynchronous operation, so we should not continue until the library is fully loaded, otherwise we could end up in a situation where we attempt to use the library before it has loaded. Use a promise here that resolves once the library is loaded.

Move the loading below the `this._bouncedEmail =`, then once loading the library has completed, call the super classes beforeRender.